src/server/parser/tokeniser.ts

```diff
@@ -456,6 +456,8 @@ export class Tokeniser
 			case 'U':
 				this.readHexToken()
 				return String.fromCodePoint(Number.parseInt(this._token.value, 16))
+			default:
+				console.warn(`Unknown unicode character: ${this.currentChar}`)
```

Comment:
This should probably generate a SyntaxError object so the system can then properly report this with an error squiggly down the road

Reply:
Ok, we can use the TODO to turn this into a SyntaxError too instead of console.warn()'ing, so we'll accept this.